### PR TITLE
doc: rewrite home page

### PR DESCRIPTION
**Pull request by QuentinBrosse**

### CODE REVIEW

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -2,120 +2,165 @@
...
-UUIDs.
+## Authentication
 
-Your **organization ID** can be found in the *Account* tab of the Scaleway control panel. It
-is labeled "Organization ID". Alternatively, if you already have a **secret token** you can
-issue a request to the Scaleway API directly and query for your **organization ID**:
-```shell
-$ curl https://account.scaleway.com/organizations -H "X-Auth-Token: <YOUR-SECRET-TOKEN>"
-```
+The Scaleway authentication is based on an **access key** and and **secret key**.
```
Comment (jerome-quere) [inline website/docs/index.html.markdown]: `and and` => `and a`

File: website/docs/index.html.markdown [hunk 2]
```diff
@@ -2,120 +2,165 @@
...
-  The Scaleway provider is used to interact with Scaleway bare metal & VPS provider.
+  The Scaleway provider is used to manage Scaleway resources. The provider needs to be configured with the proper credentials before it can be used.
 ---
 
 # Scaleway Provider
 
 The Scaleway provider is used to manage Scaleway resources.
+The provider needs to be configured with the proper credentials before it can be used.
+
+This is the documentation for the version `>= 1.11.0` of the provider. If you comes from `< v1.11.0`, checkout to migration guide.
```
Comment (DenBeke) [inline website/docs/index.html.markdown]: come ~s~

File: website/docs/index.html.markdown [hunk 3]
```diff
@@ -2,120 +2,165 @@
...
-* A security group.
+Here is an example that will setup a web server with an additional volume, a public IP and a security group.
 
-(create this as `sl.tf` and run terraform commands from this directory):
+You can test this config by creating a `test.tf` and run terraform commands from this directory:
+
+- Get your [Scaleway credentials](https://console.scaleway.com/account/credentials) 
+- Initialize a Terraform working directory: `terraform init`
+- Generate and show the execution plan: `terraform plan`
+- Builds the infrastructure: `terraform apply`
```
Comment (DenBeke) [inline website/docs/index.html.markdown]: Build ~s~

File: website/docs/index.html.markdown [hunk 4]
```diff
@@ -2,120 +2,165 @@
...
+$ export SCW_ACCESS_KEY="my-access-key"
+$ export SCW_SECRET_KEY="my-secret-key"
+$ terraform plan
+```
+
+### Shared configuration file
+
+It is a YAML configuration file shared between the majority of the
+[Scaleway developer tools](https://developers.scaleway.com/en/community-tools/#official-repos).
+Its default location is `$HOME/.config/scw/config.yaml` (`%USERPROFILE%/.config/scw/config.yaml` on windows).
```
Comment (DenBeke) [inline website/docs/index.html.markdown]: *W*indows

File: website/docs/index.html.markdown [hunk 5]
```diff
@@ -2,120 +2,165 @@
...
+If it fail to detect credentials inline, or in the environment, Terraform will check this file.
```
Comment (DenBeke) [inline website/docs/index.html.markdown]: fail**s**

File: website/docs/index.html.markdown [hunk 6]
```diff
@@ -2,120 +2,165 @@
...
+
+You can optionally specify a different location with `SCW_CONFIG_PATH` environment variable.
+You an find more information about this configuration [in the documentation](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md#scaleway-config).
```
Comment (DenBeke) [inline website/docs/index.html.markdown]: **c**an

File: website/docs/index.html.markdown [hunk 7]
```diff
@@ -2,120 +2,165 @@
...
+
+## Arguments Reference
+
+In addition to [generic provider arguments](https://www.terraform.io/docs/configuration/providers.html) (e.g. `alias` and `version`), the following arguments are supported in the Scaleway provider block:
+
+- `access_key` - (Optional) The Scaleway access key. It must be provided, but it can also be sourced from
+the `SCW_ACCESS_KEY` [environment variable](#environment-variables), or via a [shared configuration file](#shared-configuration-file),
+in this priotity order.
```
Comment (DenBeke) [inline website/docs/index.html.markdown]: prio**t**ity -> priority

File: website/docs/index.html.markdown [hunk 8]
```diff
@@ -2,120 +2,165 @@
...
 
-- **SCALEWAY_ORGANIZATION**: Your Scaleway `organization` ID
-- **SCALEWAY_TOKEN**: Your API access `token`, generated by you
-- **SCALEWAY_REGION**: The Scaleway region
+- `secret_key` - (Optional) The Scaleway secert key. It must be provided, but it can also be sourced from
+the `SCW_SECRET_KEY` [environment variable](#environment-variables), or via a [shared configuration file](#shared-configuration-file),
+in this priotity order.
```
Comment (DenBeke) [inline website/docs/index.html.markdown]: prio**t**ity -> priority

File: website/docs/index.html.markdown [hunk 9]
```diff
@@ -2,120 +2,165 @@
...
 
-Alternatively the `~/.scwrc` will be parsed to load your Scaleway CLI configuration, if it exists.
+- `project_id` - (Optional) The project ID that will be used as default value for all resources. It can also be sourced from
+the `SCW_DEFAULT_PROJECT_ID` [environment variable](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md#environment-variables), or via a [shared configuration file](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md#scaleway-config),
+in this priotity order.
```
Comment (DenBeke) [inline website/docs/index.html.markdown]: prio**t**ity -> priority

File: website/docs/index.html.markdown [hunk 10]
```diff
@@ -2,120 +2,165 @@
...
 
-## Volume usage
+- `region` - (Optional) The region that will be used as default value for all resources. It can also be sourced from
+the `SCW_DEFAULT_REGION` [environment variable](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md#environment-variables), or via a [shared configuration file](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md#scaleway-config),
+in this priotity order.
```
Comment (DenBeke) [inline website/docs/index.html.markdown]: prio**t**ity -> priority

File: website/docs/index.html.markdown [hunk 11]
```diff
@@ -2,120 +2,165 @@
...
 
-You can add volumes to bare metal instances.
-The minimal size of increment is 50GB and you can add at most 15 different volumes on an instance.
+[//]: # (TODO: Add region link to TF doc)
 
-Additional volumes cannot be added to virtual cloud servers.
+- `zone` - (Optional) The zone that will be used as default value for all resources. It can also be sourced from
+the `SCW_DEFAULT_ZONE` [environment variable](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md#environment-variables), or via a [shared configuration file](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md#scaleway-config),
+in this priotity order.
```
Comment (DenBeke) [inline website/docs/index.html.markdown]: prio**t**ity -> priority

**Pull request merged**